python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,59 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zdiff(self, keys: List[str]) -> List[str]:
+        """
+        Returns the difference between the first sorted set and all the successive sorted sets.
+        To get the elements with their scores, see `zdiff_withscores`.
+
+        When in Cluster mode, all keys must map to the same hash slot.
+
+        See https://valkey.io/commands/zdiff for more details.
+
+        Args:
+            keys (List[str]): The keys of the sorted sets.
+
+        Returns:
+            List[str]: A list of elements representing the difference between the sorted sets.
+                If the first key does not exist, it is treated as an empty sorted set, and the command returns an
+                empty list.
+
+        Examples:
+            >>> await client.zdiff("sorted_set1", "sorted_set2", "sorted_set3")
+                ["element1"]  # Indicates that "element1" is in "sorted_set1" but not "sorted_set2" or "sorted_set3".
+        """
+        return cast(
+            List[str],
+            await self._execute_command(RequestType.ZDiff, [str(len(keys))] + keys),
+        )
+
+    async def zdiff_withscores(self, keys: List[str]) -> Dict[str, float]:
+        """
+        Returns the difference between the first sorted set and all the successive sorted sets.
+        When in Cluster mode, all keys must map to the same hash slot.
+
+        See https://valkey.io/commands/zdiff for more details.
+
+        Args:
+            keys (List[str]): The keys of the sorted sets.
+
+        Returns:
+            Dict[str, float]: A dictionary of elements and their scores representing the difference between the sorted
```

Comment:
`Dict` or `Mapping`?

Reply:
I wasn't aware of the Mapping type but apparently it is a more general type than Dict. Its unclear which one is better, the existing code uses both somewhat interchangeably. According to [this SO post](https://stackoverflow.com/a/75722246/22506474) Dict is preferred for return types and Mapping for arguments, but the existing code has both types in both places